guides/team-and-project-scheduling

```diff
@@ -0,0 +1,67 @@
+# Team and project scheduling
+
+***If there are any questions about anything mentioned in this guide, put a message in the #scheduling slack channel - most people in this channel have a good level of knowledge of 10kft and should be able to offer clarification.***
+
+- [What we use 10kft for and why](## What we use 10kft for and why)
```

Comment:
Is this the first of what will eventually be many internal links in the document? I feel like it's a bit redundant as it will just create a link to the next line. I'd remove it for now.

Reply:
It'll be a list of 5 sections, pointing people to different section of the guide, i.e:

- Why we use 10kft
- If you're an admin
- If you're a team member
- If you're a scheduler
- How the support rota works

Fine to remove this for now until we've added in the rest and can see how that works best.

Reply:
Aha I see, you can leave it in if you're going to add more 👍